### ISSUE TITLE
Session HI Jacking

### ISSUE THREAD

**Issue by grahamehorner** [praeclarum/Ooui#58]
Please consider using a similar security method to SignalR to prevent session hi jacking, see https://github.com/aspnet/SignalR/issues/1155

**Comment by praeclarum** [comment]
Yes I've been meaning to add some form of security. Thanks for the link.

**Comment by grahamehorner** [comment]
DH ? Device held ? I would agree but the web socket protocol would also need to be secured, wss:// not ws://

**Comment by praeclarum** [comment]
Diffie–Hellman key exchange

Yes, #51 will fix wss://
